### PR TITLE
Allow the "action_steps" of an invalid license message to be filterable

### PR DESCRIPTION
**Pull request by borkweb**
This will help with altered messaging for various other new plugins in the works.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Expected 1 space before &quot;&lt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space after &quot;&lt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space before &quot;&gt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space after &quot;&gt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space before &quot;&lt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space after &quot;&lt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space before &quot;/&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space after &quot;/&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | Expected 1 space before &quot;&gt;&quot;; 0 found | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)
:x: | String &quot; );n	}nn	/**n	 * Generate a notice listing any plugins for which license keys have expired.n	 *n	 * This notice should only appear at the top of the plugin admin screen and &quot; does not require double quotes; use single quotes instead | [Notices.php:242](https://github.com/moderntribe/tribe-common/blob/3391e1cd4104f6816fc8f010e275c389db0de474/src/Tribe/PUE/Notices.php#L242)

> via jenkins/codesniffer-phpcs

File: src/Tribe/PUE/Notices.php [hunk 1]
```diff
@@ -227,9 +227,19 @@ public function render_invalid_key() {
...
+		/**
+		 * Filters the actions that can be taken if an invalid key is present
+		 *
+		 * @param string $actions Actions
+		 * @param array $plugin_names Plugin names the message applies to
+		 */
+		$action_steps = apply_filters( 'tribe_notice_invalid_key_actions', $this->find_your_key_text(), $plugin_names );
+
+		if ( $action_steps ) {
+			$action_steps = "<p>{$action_steps}</p>';
```
Comment (bordoni) [inline src/Tribe/PUE/Notices.php]: Starting with `"` and not closing.

**Comment by borkweb** [comment]
Fixed

**Comment by bordoni** [comment]
👍

**Pull request merged**